include/deal.II/lac/lapack_templates.h

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2005 - 2018 by the deal.II authors
+// Copyright (C) 2005 - 2021 by the deal.II authors
```

Comment:
This file is autogenerated, right?

Maybe as a follow-up, it would be useful to not check it into the repository but in fact generate it on the fly in the build directory.

Reply:
It is autogenerated but this is problematic for two reasons:
1. 99% of the data we use to generate it comes from an external copy of LAPACK, where we parse their function declarations to work out types and constness. Hence we can't generate this file unless someone downloads LAPACK and points `generate_lapack_bindings` to that source directory
2. The script we use to generate it is written in Python (though it wouldn't be too hard to convert it to C++)

Reply:
I think we already use python for some of our scripts. For sure we use PERL to transform the `.inst.in` files.

Reply:
What? We use `expand_instantiations.cc`. I don't think we have any build dependencies on perl except for the documentation.

Reply:
I stand corrected :-)

Reply:
I think we accidentally acquired a python dependency at some point and it was a minor disaster. We need to keep these things optional for building the library itself.